models/public/fcrn-dp-nyu-depth-v2-tf/fcrn-dp-nyu-depth-v2-tf.md

```diff
@@ -0,0 +1,111 @@
+# fcrn-dp-nyu-depth-v2-tf
+
+## Use Case and High-Level Description
+
+FCRN ResNet50 UpProj is a model for monocular depth estimation trained by NYU Depth V2 dataset,
+  as described in the paper [Deeper Depth Prediction with Fully Convolutional Residual Networks](https://arxiv.org/abs/1606.00373). 
+  The model input is a blob that consists of a single image of "1x228x304x3" in RGB order.
```

Comment:
```suggestion
  The model input is a single color image.
```

I think this is less confusing, because the dimensions and the channel order are different between the original and converted models.